.gitignore

```diff
@@ -2,33 +2,353 @@
 /build
 AssemblyInfo.cs
```

Comment:
We can remove the AssemblyInfo.cs exclusion now, we used to generate these with the old tooling.